scripts/add-github-sourcelink.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+###################################################################################################
+# add-github-sourcelink                                                                           #
```

Comment:
It's nice that it's a generic script (though it needn't be) but then the name and docs need to reflect it.